radialgraph/src/main/java/com/duartbreedt/radialgraph/model/Section.kt

```diff
@@ -9,29 +9,34 @@ class Section {
     val value: BigDecimal
     @ColorInt
     val color: Int
+    @ColorInt
+    val gradientColors: List<Int>?
```

Comment:
Please rather make `val color: Int` into `val color: List<Int>` and remove `val gradientColors: List<Int>?`